lib/rational/include/Rational.h

```diff
@@ -1,33 +1,36 @@
 #pragma once
 
-
+#include <iostream>
 
 class Rational {
 public:
-	Rational();
+	Rational() = default;
 	Rational(int numerator, int denominator);
 
-	Rational operator+(const Rational& other);
-	Rational operator-(const Rational& other);
-	Rational operator*(const Rational& other);
-	Rational operator/(const Rational& other);
+	Rational operator+=(const Rational other);
+	Rational operator-=(const Rational other);
+	Rational operator*=(const Rational other);
+	Rational operator/=(const Rational other);
+
+	double ToDouble();
 
-	double toDouble();
+	friend std::ostream& operator<<(
+		std::ostream& os,
+		const Rational
+	);
 
-	friend bool operator==(const Rational& left, const Rational& right);
+	friend bool operator==(const Rational lhs, const Rational rhs);
+	friend bool operator!=(const Rational lhs, const Rational rhs);
 
 private:
-	Rational reduce() const;
+	auto Print(std::ostream& os) const ->std::ostream&;
+	Rational Reduce() const;
 
-	int numerator = 1;
-	int denominator = 1;
+	int numerator = { 1 };
+	int denominator = { 1 };
```

Comment:
Use prefix or postfix for members names

Reply:
What do you have in mind?

Reply:
Members should be named like this
m_numerator or numerator_;